recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
++ **First cooked by:**
+[@Karyum](https://github.com/karyum)
+
++ **Date:** Week 5 FACN2
++ **Author of Recipe:**
+#MrsSaliba
+
+### Ingredients
+- 2 small eggs.
+- a handful of parsley.
+- a handful of green/spring onions.
+- zaatar to taste.
```

Comment:
How would I know the amount the first time I make this. You mix everything together and I would rather not taste a raw egg mixture to check the taste